src/Plugins/Nop.Plugin.ExternalAuth.Facebook/Controllers/FacebookDataDeletionController.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Facebook;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Newtonsoft.Json;
+using Nop.Core;
+using Nop.Plugin.ExternalAuth.Facebook.Models;
+using Nop.Services.Authentication.External;
+using Nop.Services.Localization;
+using Nop.Services.Logging;
+using Nop.Services.Messages;
+using Nop.Web.Framework.Controllers;
+
+namespace Nop.Plugin.ExternalAuth.Facebook.Controllers
+{
+    public class FacebookDataDeletionController : BasePluginController
```

Comment:
Need to use `Microsoft.AspNetCore.Mvc.Controller` here and leave Post request only.